pcmdi_metrics/precip_variability/lib/lib_variability_across_timescales.py

```diff
@@ -230,7 +214,7 @@ def CropLatLon(d, regions_specs):
     """
     region_name = list(regions_specs.keys())[0]
 
-    dnew = region_subset(d, regions_specs, region=region_name)
+    dnew = region_subset(d, region_name, None, regions_specs, False)
```

Comment:
You might want to double check this with the function definition at https://github.com/PCMDI/pcmdi_metrics/blob/30640f10064d4bd8b1a4ee649cef76ae5fe2ffb3/pcmdi_metrics/io/regions.py#L82

My suggestion for this line for more clarification:

`dnew = region_subset(d, region_name, regions_specs=regions_specs)`